containers-sig-laws/app/Main.hs

```diff
@@ -0,0 +1,77 @@
+{-# LANGUAGE TypeApplications #-}
+
+module Main where
+
+import           Test.QuickCheck
```

Comment:
Could you, please, make this import implicit? This will help in future 🙂

Reply:
Yes, sorry for not being clear enough. I mean _explicit import list_. We follow our style guide and write all exports and imports with explicit lists. This requires more time but helps code readability a lot 👍